src/improvement-2.js

```diff
@@ -4,7 +4,6 @@
  *  PLEASE DO NOT REMOVE `export default function`
  */
 
-// FUNCTION 2
 const onlyGetFoods = (items)=> {
```

Comment:
**Optional**: Can you consider implicitly return as you are using arrow function with single expression to benefit shorter syntax and clean codes?